test/powershell/engine/ExperimentalFeature/Get-ExperimentalFeature.Tests.ps1

```diff
@@ -173,6 +173,8 @@ Describe "Default enablement of Experimental Features" -Tags CI {
     It "On preview builds, Experimental Features are enabled" -Skip:(!$isPreview) {
         (Join-Path -Path $PSHOME -ChildPath 'powershell.config.json') | Should -Exist
 
+        Get-ExperimentalFeature | fl *  | Out-String | Write-Verbose -Verbose
+
         foreach ($expFeature in Get-ExperimentalFeature) {
             if ($expFeature.Name -ne "PS7DscSupport")
```

Comment:
Is there a need for experimental feature for DSCV3?

Reply:
This was actually discussed offline with Dongbo and Steve during subsystem work.
In short the answer is - no need for experimental feature because loading/unloading of the module (equivalent of enabling/disabling exp.feature) will signal user's intent to use v3 features.